src/mainwindow.cpp

```diff
@@ -1149,9 +1167,9 @@ void GeQueryInfoViz(TreeModel* model)
                     {
                         // Hyper does not use square brackets for table names, but the table names
                         // we log in sql-temp-table events have them.
-                        tableName = "[" + tableName + "]";
+                        tableName = "[" % tableName % "]";
                     }
-                    outputFedTempTable += queryHash + ",\"" + tableName + "\"\n";
+                    outputFedTempTable += queryHash % ",\"" % tableName % "\"\n";
```

Comment:
That uses QStringBuilder implicitly, [so it concatenates more efficient](https://doc.qt.io/qt-5/qstring.html#more-efficient-string-construction).
It does not have a %= operator though.